eclair-core/src/test/scala/fr/acinq/eclair/channel/states/StateTestsHelperMethods.scala

```diff
@@ -281,14 +281,17 @@ trait StateTestsHelperMethods extends TestKitBase {
 
   def localClose(s: TestFSMRef[State, Data, Channel], s2blockchain: TestProbe): LocalCommitPublished = {
     // an error occurs and s publishes its commit tx
-    val commitTx = s.stateData.asInstanceOf[DATA_NORMAL].commitments.localCommit.publishableTxs.commitTx.tx
+    val commitTx = s.stateData.asInstanceOf[DATA_NORMAL].commitments.localCommit.commitTxAndRemoteSig.commitTx.tx
```

Comment:
I think this function is the right place to verify that our txs are stored unsigned. We could add here:
```scala
commitTx.txIn.foreach(txIn => assert(txIn.witness.isNull))
```

And below, the same thing for HTLC txs?

Reply:
Done in 2e074f710 (squashed).